src/helix/client/client_ext.rs

```diff
@@ -12,42 +12,45 @@ impl<'a, C: crate::HttpClient<'a> + Sync> HelixClient<'a, C> {
     /// Get [User](helix::users::User) from user login
     pub async fn get_user_from_login<T>(
         &'a self,
-        login: impl Into<types::UserName>,
+        login: impl Into<&types::UserNameRef>,
```

Comment:
These should be `AsRef<types::UserNameRef>` etc

same applies for all of the `client_ext.rs` functions

Reply:
I previously had this as `AsRef<types::UserNameRef>`, but it's a bit inconvenient, since only `types::UserName` implements `AsRef<types::UserNameRef>` thus you cannot call this with a `&str`. But I can retry it.

Reply:
oh I see! im thinking of the old `aliri_braid` or just thinking in reverse. nevermind :)